File: app/src/main/java/com/MohamedTaha/Imagine/New/mvp/model/azan/Timings.java

package com.MohamedTaha.Imagine.New.mvp.model.azan;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.Ignore;
import androidx.room.PrimaryKey;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
@Entity(tableName = "prayer_time")
public class Timings {
    public Timings() {
    }
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getDate_today() {
        return date_today;
    }

    public void setDate_today(String date_today) {
        this.date_today = date_today;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getNote() {
        return note;
    }

    public void setNote(String note) {
        this.note = note;
    }

    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "id_prayer_time")
    private int id;


    private String date_today;
    private String city;
    private String note;
    @SerializedName("Fajr")
    @Expose
    private String fajr;
    @SerializedName("Sunrise")
    @Expose
    private String sunrise;
    @SerializedName("Dhuhr")
    @Expose
    private String dhuhr;
    @SerializedName("Asr")
    @Expose
    private String asr;
    @SerializedName("Sunset")
    @Expose
    private String sunset;
    @SerializedName("Maghrib")
    @Expose
    private String maghrib;
    @SerializedName("Isha")
    @Expose
    private String isha;
    @SerializedName("Imsak")
    @Expose
    private String imsak;
    @SerializedName("Midnight")
    @Expose
    private String midnight;
    private int id_seq;
    public int getId_seq() {
        return id_seq;
    }

    public void setId_seq(int id_seq) {
        this.id_seq = id_seq;
    }

    public String getDay_today_hegry() {
        return day_today_hegry;
    }

    public void setDay_today_hegry(String day_today_hegry) {
        this.day_today_hegry = day_today_hegry;
    }

    private String day_today_hegry ;

    public String getFajr() {
        return fajr;
    }

    public void setFajr(String fajr) {
        this.fajr = fajr;
    }

    public String getSunrise() {
        return sunrise;
    }

    public void setSunrise(String sunrise) {
        this.sunrise = sunrise;
    }

    public String getDhuhr() {
        return dhuhr;
    }

    public void setDhuhr(String dhuhr) {
        this.dhuhr = dhuhr;
    }

    public String getAsr() {
        return asr;
    }

    public void setAsr(String asr) {
        this.asr = asr;
    }

    public String getSunset() {
        return sunset;
    }

    public void setSunset(String sunset) {
        this.sunset = sunset;
    }

    public String getMaghrib() {
        return maghrib;
    }

    public void setMaghrib(String maghrib) {
        this.maghrib = maghrib;
    }

    public String getIsha() {
        return isha;
    }

    public void setIsha(String isha) {
        this.isha = isha;
    }

    public String getImsak() {
        return imsak;
    }

    public void setImsak(String imsak) {
        this.imsak = imsak;
    }

    public String getMidnight() {
        return midnight;
    }

    public void setMidnight(String midnight) {
        this.midnight = midnight;
    }

}

File: app/src/main/java/com/mohamedtaha/imagine/mvp/model/azan/Timings.kt
package com.mohamedtaha.imagine.mvp.model.azan

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

@Entity(tableName = "prayer_time")
class Timings {
    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "id_prayer_time")
    var id = 0
    var date_today: String? = null
    var city: String? = null
    var note: String? = null

    @SerializedName("Fajr")
    @Expose
    var fajr: String? = null

    @SerializedName("Sunrise")
    @Expose
    var sunrise: String? = null

    @SerializedName("Dhuhr")
    @Expose
    var dhuhr: String? = null

    @SerializedName("Asr")
    @Expose
    var asr: String? = null

    @SerializedName("Sunset")
    @Expose
    var sunset: String? = null

    @SerializedName("Maghrib")
    @Expose
    var maghrib: String? = null

    @SerializedName("Isha")
    @Expose
    var isha: String? = null

    @SerializedName("Imsak")
    @Expose
    var imsak: String? = null

    @SerializedName("Midnight")
    @Expose
    var midnight: String? = null
    var id_seq = 0
    var day_today_hegry: String? = null

    override fun equals(other: Any?): Boolean {
        return super.equals(other)
    }
}